flake8_quotes/__init__.py

```diff
@@ -77,15 +77,22 @@ def get_quotes_errors(self, file_contents):
                 # ignore non strings
                 continue
 
-            if not token.string.startswith(self.inline_quotes['bad_single']):
+            # Remove any prefixes in strings like `u"foo"`. Use the quote
+            # character at the end of the string to know where the prefixes end
+            quote_char = token.string[-1]  # Does only contain one character!
+            quote_start = token.string.index(quote_char)
+            text = token.string[quote_start:]
+            # `"foo"`, `b"foo"`, `br"foo"` all result in `"foo"`
```

Comment:
This should likely go at the head with the rest of the comments and we should mention that it's `text's` value but to avoid wasting even more time I will land this + make the minor edits.